build.gradle

```diff
@@ -118,6 +118,8 @@ dependencies {
     }
     compile 'org.springframework.hateoas:spring-hateoas:0.19.0.RELEASE'
 
+    compile 'org.eclipse.jetty:jetty-servlets:9.4.0.M0'
```

Comment:
Would be safer to use the same version as other Jetty dependencies used in the Scheduling project. Version 9.4.0 looks pretty recent and may cause some problems when the Workflow Catalog is included the PWS distribution.